content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md

```diff
@@ -37,6 +37,14 @@ Each CODEOWNERS file assigns the code owners for a single branch in the reposito
 
 For code owners to receive review requests, the CODEOWNERS file must be on the base branch of the pull request. For example, if you assign `@octocat` as the code owner for *.js* files on the `gh-pages` branch of your repository, `@octocat` will receive review requests when a pull request with changes to *.js* files is opened between the head branch and `gh-pages`.
 
+## CODEOWNERS and forks
+
+Asides being used to tell at a glance who is responsible for a given file, CODEOWNERS are also used for triggering review requests for a pull request that modifies an owned file.
+
+Pull requests use the version of CODEOWNERS from the base branch of a pull request. A base branch is the branch that a pull request will modify if the pull request is merged. If the repo that you forked from is the base of your PR, then the PR will use the CODEOWNERS file from the repo that you forked from. If your PR is targeting a branch within your forked repo, then the PR will use the CODEOWNERS on that base branch in your forked repo.
+
+When seeing who responsible for a file, the CODEOWNERS for whatever branch you're looking at on whatever repo you're looking at is used.
```

Comment:
Please help

Reply:
`seeing` definitely doesn't work. I don't think `When you view` works great either. Even though this file is about `CODEOWNERS`, I think it's better to say something like:

```
When you want to find accounts responsible for a file,
you can find that information in the `CODEOWNERS` file
for whichever branch in whichever repository you're looking at.
```

Reply:
So just to clarify, this paragraph is referring to what you see when you hover over the little shield icon for a file. 

<img width="558" alt="CleanShot 2023-06-02 at 09 48 29@2x" src="https://github.com/github/docs/assets/101839405/2874f15f-f620-42d6-b4d1-d2a30d352781">

It would actually be good to specify that, @OlayinkaAtobiloye. Something like this?

> When you view who is responsible for a file by hovering over {% octicon "shield-lock" aria-label="Owned by USER or TEAM (from CODEOWNERS line NUMBER)" %}, you will see information from the `CODEOWNERS` file for whichever branch in whichever repository you're looking at.